README.md

```diff
@@ -216,6 +216,87 @@ stripe.setAppInfo({
 
 This information is passed along when the library makes calls to the Stripe API.
 
+### Auto-pagination
+
+As of stripe-node 6.11.0, you may auto-paginate list api methods.
```

Comment:
Can you capitalize to "API" given this is user facing?

Reply:
hmm, actually I think I'll just remove the word since it's used differently than the next sentence: `you may auto-paginate list methods`. sgty?

Reply:
Yep! SG.